### ISSUE TITLE
Brew uses and brew leaves ignore R deps

### ISSUE THREAD

**Issue by CamJN** [Homebrew/legacy-homebrew#48378]
I have `R` installed, it depends on `gcc`, and homebrew knows this because `brew deps r` includes `gcc`, but when I run `brew uses gcc --installed` or `brew leaves` that dep is ignored. I'm not 100% sure where the issue lies, but I reported it to the homebrew-science tap and was directed to report it here.

`r` depends on `gcc` because of its `:fortran` dependency, and perhaps also indirectly via other dependencies.

**Comment by apjanke** [comment]
I can reproduce, on my 10.9 box. And yeah, this is the right place to report it.

Dunno what's going on here. The `uses --installed` seems to be producing too little output.

```
[~]
$ brew uses gcc --installed
homebrew/python/numpy                    homebrew/science/openblas
```

I've got a lot of stuff installed, and it seems like more should be picking up `gcc` through that `:fortran` requirement. I checked the `openblas` formula, and it's a `:fortran` requirement there, too, that's pulling in `gcc`: that's its [only `depends_on`](https://github.com/Homebrew/homebrew-science/blob/13b4f4f914ddac583291bc4e7648d831fc9e62af/openblas.rb#L14).

Would be nice if something like this worked, too.

```
$ brew uses :fortran
Error: No available formula with the name ":fortran"
```
